components/EditCoverSection.tsx

```diff
@@ -87,7 +87,48 @@ const EditCoverSection = () => {
     }
   };
 
+  const imageRef = React.useRef<any>(null);
+
+  const handleViewClick = () => {
+    if (imageRef?.current instanceof HTMLImageElement) {
+      imageRef.current.click();
+    }
+  };
+
   const handleView = (e: any) => {
+    const imageUrl = e.target.src;
+    console.log(e.target);
+    const lightbox = document.createElement("div");
+    lightbox.style.position = "fixed";
+    lightbox.style.top = "0";
+    lightbox.style.bottom = "0";
+    lightbox.style.left = "0";
+    lightbox.style.right = "0";
+    lightbox.style.background = "rgba(0,0,0,0.8)";
+
+    const image = document.createElement("img");
+    image.style.position = "absolute";
+    image.style.top = "50%";
+    image.style.left = "50%";
+    image.style.transform = "translate(-50%, -50%)";
+    image.src = imageUrl;
+    lightbox.appendChild(image);
+
+    const closeButton = document.createElement("button");
+    closeButton.style.position = "absolute";
```

Comment:
Instead of adding multiple lines for style we can create a class or component in our global.css